src/search.js

```diff
@@ -10,34 +10,56 @@ const critterTypes = {
 
 const sortBy = (filter) => R.sortBy(R.prop(filter));
 
-const getLeavingNow = ({ baseResults, hemisphere }) => {
-  const currentMonth = new Date().getMonth() + 1; // because javascript thinks jan is 0
-  return baseResults.filter(
-    (item) =>
-      item.available[hemisphere].includes(currentMonth) &&
-      !item.available[hemisphere].includes(currentMonth + 1)
-  );
+const filterEverything = ({
```

Comment:
I wanted to use a Ramda function for this, I still might, but this is what it is right now.

Reply:
Haha, just saw this. Glad to see I'm not the only one who Ramda baffles. :)

Reply:
Honestly, I think the way you have it is plenty readable. If you wanted, you could bump up the readability maybe a tad more by including a quick `//` comment above each section, e.g.:

```js
// Filter by the searched text
if (searchText) {
  ...
}

// Filter by "Leaving this month"
if (leavingNow) {
  ...
}

// Filter by month
if (!leavingNow && monthsToFilter && monthsToFilter.length > 0) {
  ...
}
```

But again, I honestly think it is fine as-is.